src/shell/shell.cpp

```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
 	        "REN [drive:][path]filename1 filename2.\n\n"
 	        "Note that you can not specify a new drive or path for your destination file.\n");
 	MSG_Add("SHELL_CMD_DELETE_HELP","Removes one or more files.\n");
+	MSG_Add("SHELL_CMD_DELETE_HELP_LONG", "Usage:\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "  \033[32;1merase\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPATTERN\033[0m is a file name or pattern that contains wildcards (* or ?),\n"
+	        "          such as \033[36;1mfile.txt\033[0m, or \033[36;1m*.bat\033[0m.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  Be careful with deleting files containing wildcards, especially \033[36;1m*.*\033[0m.\n"
+            "  All files matching the pattern(s) will be deleted.\n"
```

Comment:
This line is indented with only spaces (instead of one tab and then aligned with spaces).

Reply:
Thanks for catching this single line. Fixed.